src/patterns/dosdp-patterns/disease_series_by_translocation.yaml

```diff
@@ -0,0 +1,55 @@
+pattern_name: disease_series_by_translocation
+
+description: >-
+  This pattern will be used for diseases that are caused by a specific chromosomal translocation. The content below is just a placeholder, as the class to be used in the variable is to be determined. 
```

Comment:
remember the pattern for patterns. This is too general. It makes it sound like any subclass of chromosomal disease should have this. 

In fact it is *only* for disease *classes* that are of the form "{translocation} disease"

Reply:
oh I take it back. Actually we need two patterns

 - D caused by T
 - any disease caused by T

also is it only translocations?